hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OzoneAclUtil.java

```diff
@@ -88,17 +89,18 @@ public static List<OzoneAcl> filterAclList(String identityName,
     return retList;
   }
 
-  private static boolean checkAccessInAcl(OzoneAcl a, String[] groups,
-      String username, ACLType aclToCheck) {
+  private static boolean checkAccessInAcl(OzoneAcl a, UserGroupInformation ugi,
+      ACLType aclToCheck) {
     BitSet rights = a.getAclBitSet();
     switch (a.getType()) {
     case USER:
-      if (a.getName().equals(username)) {
+      if (a.getName().equals(ugi.getShortUserName()) || a.getName()
+          .equals(ugi.getUserName())) {
```

Comment:
I feel like it should stick with getShortUserName() and that it's the caller who creates the OzoneAcl objects should make sure it passes in short user names.

Reply:
make sense.